pages/common/kdeconnect-cli.md

```diff
@@ -7,21 +7,21 @@
 
 `kdeconnect-cli --list-devices`
 
-- List all reachable devices:
+- List available (paired and reachable) devices:
 
 `kdeconnect-cli --list-available`
 
 - Request pairing with a specific device, specifying its ID:
 
 `kdeconnect-cli --pair --device {{device_id}}`
 
-- Request pairing with a specific device, specifying its name:
+- Ring a device, specifying its name:
```

Comment:
```suggestion
- Ring the specified device:
```

Reply:
I think `specifying its name` is important here, since it underlines the difference between `--device` and `--name`.

Reply:
Ah, because `--device` takes an ID instead? Then maybe to clarify:
```suggestion
- Ring a device by name rather than ID:
```
?

Reply:
I'm fine with both variants. They seem pretty much the same to me.

Reply:
@bl-ue i think adding "rather than ID" is unnecessary and could led to confusion.

Reply:
Let's skip it then. 👍🏻